devU-client/src/components/pages/assignmentFormPage.tsx

```diff
@@ -70,16 +76,27 @@ const AssignmentCreatePage = () => {
     return(
         <PageWrapper>
             <h1>Assignment Form</h1>
-            <TextField id = 'courseId' label = 'Course Id' onChange={handleChange}/>
             <TextField id='name' label='Assignment Name' onChange={handleChange}/>
```

Comment:
Did not add '*' for required field

Reply:
I thought this would be a form validation problem. For example, it says nothing when the form first appears but when the user tries to submit with empty fields, a red border around the box and 'required' would show up.

Reply:
From what Alan says, as an user, he wants to know what field is required when seeing this page for the first time instead of getting errors and lightlight the require field after submit

Reply:
Added the asterisk. Only problem is that the form won't actually stop users from submitting empty inputs which is what I thought the form validation would do.